yarn.lock

```diff
@@ -10374,8 +10374,8 @@ indexes-of@^1.0.1:
   integrity sha1-8w9xbI4r00bHtn0985FVZqfAVgc=
 
 "inera-core-css@git+https://bitbucket.org/ineraservices/inera-core-css.git#master":
-  version "0.16.0"
-  resolved "git+https://bitbucket.org/ineraservices/inera-core-css.git#75f597de378ebb76d95ed2d5b4ecd649d8e57bf2"
+  version "0.17.0"
```

Comment:
There was issues with this version, the pipeline failed after updating last time.

Reply:
Ok, I fix!